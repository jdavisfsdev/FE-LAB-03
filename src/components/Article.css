article {
    display: flex;
    flex-direction: column;
    align-items: center;
    word-wrap: break-word;
    width: 300px;
}