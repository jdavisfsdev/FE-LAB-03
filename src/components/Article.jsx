import React from 'react';
import PropTypes from 'prop-types';

export default function Article({ title, author, description }) {
  return (
    <>
      <h2>{title}</h2>
      <h3>{author}</h3>
      <p>{description}</p>
            
    </>
  );
}

Article.propTypes = {
  title: PropTypes.string.isRequired,
  author: PropTypes.string.isRequired,
  description: PropTypes.string.isRequired
};
