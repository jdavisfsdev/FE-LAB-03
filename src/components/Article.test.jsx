import React from 'react';
import { render } from '@testing-library/react';
import Article from './Article';

describe('Article component', () => {
  it('renders Article', () => {
    const { asFragment } = render (
      <Article
        title="Mama the dog"
        author="Me"
        description="need i say more?" />
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
