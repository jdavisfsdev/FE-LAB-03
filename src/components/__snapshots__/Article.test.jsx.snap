// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Article component renders Article 1`] = `
<DocumentFragment>
  <h2>
    Mama the dog
  </h2>
  <h3>
    Me
  </h3>
  <p>
    need i say more?
  </p>
</DocumentFragment>
`;
